src/common/transformations/src/transformations/op_conversions/convert_topk11_downgrade.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_topk11_downgrade.hpp"
+
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+#include <openvino/opsets/opset11.hpp>
+#include <openvino/opsets/opset3.hpp>
+
+#include "itt.hpp"
+
+ov::pass::ConvertTopk11ToTopk3::ConvertTopk11ToTopk3() {
+    MATCHER_SCOPE(ConvertTopk11ToTopk3);
+
+    const auto topk_v11_pattern = pattern::wrap_type<opset11::TopK>();
+
+    const matcher_pass_callback callback = [=](pattern::Matcher& m) {
+        const auto topk_v11 = std::dynamic_pointer_cast<opset11::TopK>(m.get_match_root());
+        if (!topk_v11 || topk_v11->get_stable()) {
```

Comment:
To enable [callback mechanism](https://github.com/openvinotoolkit/openvino/blob/91b9675bed5355a906eb63d09583e39f71662cd5/src/core/include/openvino/pass/pass_config.hpp#L101-L107):
```suggestion
        if (!topk_v11 || topk_v11->get_stable() || transformation_callback(topk_v11)) {
```